libr/util/log.c

```diff
@@ -2,30 +2,30 @@
 
 #include <r_util.h>
 
-// XXX: This api is kinda ugly.. we need to redefine it
+R_API void r_log(RLogLevel level, const char *fmt, ...) {
```

Comment:
@radare I have removed the other logs functions because not used. I think this log facility can be improved in a following PR.

Reply:
e.g. adding the possibility to redirect output to file, setting a filter on the level, etc. etc.